### PR TITLE
Added follow-unfollow from lens_v2

### PR DESCRIPTION
**Pull request by 0xsarvesh**
Added follow & unfollow events for lens-v2

Screen Shot 2023-10-19 at 2 35 37 PM
Screen Shot 2023-10-19 at 2 35 49 PM

### CODE REVIEW

**Comment by Hrishikesh-Thakkar** [comment]
How did you get these blocks? By running on Test Net?

File: lens-v2-follower-tracker/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        .lpp_contract_address
+        .expect("lpp_contract_address is already verified");
+    let mut follow_events: Vec<Follow> = vec![];
+    let mut unfollow_events: Vec<Unfollow> = vec![];
+
+    log::info!("block {}", blk.number);
+    // below block will process the follow related events
+    for log in blk.logs() {
+        let tx_hash = Hex(log.receipt.transaction.hash.clone()).to_string();
+        let addres: String = format_with_0x(Hex(log.address().clone()).to_string());
```
Comment (Hrishikesh-Thakkar) [inline lens-v2-follower-tracker/src/lib.rs]:
```suggestion
        let address: String = format_with_0x(Hex(log.address().clone()).to_string());
```

**Review by Hrishikesh-Thakkar — APPROVED** [review]

**Review by sharathkrml — APPROVED** [review]
LGTM

**Comment by 0xsarvesh** [comment]
> How did you get these blocks? By running on Test Net?

Tested on mumbai.

File: lens-v2-follower-tracker/src/lib.rs (hunk 1 above)
Comment (0xsarvesh) [reply lens-v2-follower-tracker/src/lib.rs]: Done

**Pull request merged**